client/app/components/_languageChooser/languageChooser.controller.js

```diff
@@ -2,12 +2,15 @@ class LanguageChooserController {
   constructor($translate) {
     "ngInject";
     Object.assign(this, {
-      $translate
+      $translate,
+      activeLang: $translate.use(),
```

Comment:
This would not get the updated value if the language is changed from elsewhere.

Probably find for now, but could create a service that contains a bindable object value (like the `CurrentGroup` service) that gets updated on the `$translateChangeSuccess` event (https://angular-translate.github.io/docs/#/guide/18_events) and would always contain the correct value.